PlayBooks/Official/Version-1/Large-Company/en/OpenChain Playbook - Large Company.md

```diff
@@ -0,0 +1,192 @@
+
+# Planning Our Strategy
+
+We will use the OpenChain self-certification questionnaire as a way to check our current status versus the status required for OpenChain ISO/IEC 5230:2020 self-certification. This involves understanding the:
+
+1. 	Current policy for managing decisions around open source (and other) software.
+2.	 Current processes being used for managing open source (and other) software.
+3.	 Current records showing the usage of this software over the previous year.
+
+# Executing Our Plan
+
+## Step 1
+
+Identify a key sponsor for the project. Ideally, this will be a senior executive (C-suite) with ability to drive the activities, provide repeated vocal support and assist with budget needs.
+
+Determine the scope of the initial work. This may be targeting a single product line, or a particular development team, a complete business unit or perhaps even the whole company at once. 
+
+Form a team to carry out investigations.If the initial scope is wide, the team will need to incorporate representatives from across the targeted organization.
+
+## Step 2
+
+Carry out a survey of products produced by the target organization. Talk to representatives in the development teams, sales teams and finance teams, to gain an understanding of what is under development, what is being actively sold, what is being maintained, and what is still selling and shipping, even though no-one is working on it anymore.
+
+For each product identified, also survey the open source used in the product (if any), and the processes and policies used for developing, building and shipping the product, in as far as they relate to open source usage. Note compliance tools in use, if any.
+
+Produce a report enumerating the products identified.
+
+## Step 3
+
+Review the processes and policies identified by the survey, and compare them for consistency. Compare the processes and policies against OpenChain to identify gaps. Produce a report identifying the findings of the survey and comparisons, indicating where there are policy differences, process differences, and gaps in relation to OpenChain.
+
+## Step 4
+
+Revise the scope of the initial project. It may be decided that a specific subset of the surveyed target area will be the initial scope for implementation. Define the timeline for remediating the policies and processes used within the revised scope.
+
+Define the initial needs for the OSPO: structure, staffing, tooling, budget. Determine where the OSPO will live within the company.
+
+Gain buy-in from the senior sponsor, and gain approval for the budget and staffing.
+
+Hire the OSPO leader and initial staff.
+
+## Step 5
+
+The OSPO produces revised policies and processes, and works with the product teams for the revised scope to implement the new processes.
+
```

Comment:
Process implementation is quite variable in companies -- some have central process operations, some do not.  I would take out "product teams" and just say "appropriate process group"